docs: cleanup (#336)
bareos/bareos: docs/manuals/source/DeveloperGuide/api.rst

    *.defaults job=BackupClient1
    job=BackupClient1pool=Incrementalmessages=Standardclient=client1.example.com-fdstorage=Filewhere=level=Incrementaltype=Backupfileset=SelfTestenabled=1catalog=MyCatalog

> This mode has been introduced by BAT and is now deprecated.

Reviewer: ```suggestion
This mode was introduced by BAT and is now deprecated.
```
